README.md

```diff
@@ -65,11 +65,10 @@ We expect that contributors interested in Splice may contribute, for example, ex
 Any group operating a Canton synchronizer may accept and implement new versions of the Splice reference applications at its own discretion. The amulet configuration used by any Canton synchronizer operating group can be made publicly available (visible) by that operating group. <br>
 
 ## Architecture
-The following diagrams show how the Splice applications interact with the Daml platform from Digital Asset.<br>
+The following diagram shows how the Splice applications interact with the Daml platform from Digital Asset.<br>
 
-![SV Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-1.jpg)
+![SV Node Architecture](./images/Splice-Canton-Global-Synchronizer.jpg)
```

Comment:
Did you mean to also include this new image? I don't see it in this PR.

Reply:
I did mean to include it; the image goes in a separate folder. I don't know the way in this UI to add the image in the same PR.